### PR TITLE
feat: include yaml files in generated folder

### PR DESCRIPTION
**Pull request by bayological**
**Description**

This PR adds the subgraph YAML files for each network to make it easier to trace the contracts being indexed by the subgraph. The build generates these files, similar to the broadcast files we keep in the deployment repo.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     "mainnet:codegen": "npx graph codegen ./generated/mento.mainnet.subgraph.yaml",
     "mainnet:build": "npx graph build ./generated/mento.mainnet.subgraph.yaml",
     "mainnet:build-all": "npm run clean && npm run mainnet:compile && npm run mainnet:codegen && npm run mainnet:build",
-    "mainnet:deploy": "npx graph deploy --studio mento ./generated/mento.mainnet.subgraph.yaml"
+    "mainnet:deploy": "npx graph deploy --studio mento-celo ./generated/mento.mainnet.subgraph.yaml"
```
Comment (chapati23) [inline package.json:19]: have we documented anywhere how to access the subgraph? i remember we (temporarily?) lost access to the first one, then I set up another one from the dev wallet, now this is the 3rd one I see

would be good to update the respective README section https://github.com/mento-protocol/mento-subgraph/?tab=readme-ov-file#how-do-i-access-the-subgraph-dashboard-on-thegraphcom

**Review by chapati23 — COMMENTED** [review]
looks good, just 1 comment on subgraph access

File: package.json (hunk 1 above)
Comment (bayological) [reply package.json:19]: > have we documented anywhere how to access the subgraph? i remember we (temporarily?) lost access to the first one, then I set up another one from the dev wallet, now this is the 3rd one I see
> 
> would be good to update the respective README section https://github.com/mento-protocol/mento-subgraph/?tab=readme-ov-file#how-do-i-access-the-subgraph-dashboard-on-thegraphcom

Nothing has changed regarding how we access the subgraph; we will still use the dev wallet. When we set up a safe that we can use for shared access, we will update to use that.

**Review by chapati23 — APPROVED** [review]
les goh

**Pull request merged**